src/components/MenuPanel/MenuPanel.js

```diff
@@ -256,6 +315,18 @@ AnimatedMenuPanel.propTypes = {
   onCloseMenuPanel: PropTypes.func.isRequired,
 }
 
+const StyledButton = styled.button`
```

Comment:
Just in case you don’t know about it, [`ButtonBase`](https://github.com/aragon/aragon-ui/blob/master/src/components/Button/ButtonBase.js) should give you the base styles + the focus ring out of the box.

Reply:
Great, then no need to add those manually.